### DISCUSSION TITLE
节点钱包转账，特定数量的时候报错

### DISCUSSION THREAD

**Discussion by jack8086wang** [nervosnetwork/ckb#2417] [General]
用节点钱包转账，某些特定的数量会提示：

**Comment by jack8086wang** [comment]
Transaction fee can not be more than 1.0 CKB, please change to-capacity value to adjust
比如：5752这个数字会报错，换成571xx-576xx都会，577xx就不会。

**Comment by quake** [comment]
你是用 ckb-cli 命令来转账的？给的参数 `tx-fee` 手续费如果大于 1 CKB 了，就会提示出这个错误，请检查一下是不是给了一个错误不合理的超大手续费。

**Comment by jack8086wang** [comment]
那肯定tx-fee设置肯定是小于1的， 我默认用的是tx-fee  0.0005

**Comment by doitian** [comment]
可能是找零不足以创建一个 change cell，然后只能当成手续费了。

@TheWaWaR 确认下，如果是这样的话需要怎么规避？如果有足够的 live cells，ckb-cli 会自动再去找一个 input 来凑吗？
